src/workerd/io/compatibility-date.capnp

```diff
@@ -352,4 +352,12 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # In the WebCrypto API, the `publicExponent` field of the algorithm of RSA keys would previously
   # be an ArrayBuffer. Using this flag, publicExponent is a Uint8Array as mandated by the
   # specification.
+
+  jsRpcStub @37 :Bool
+      $compatEnableFlag("durable_object_js_rpc")
```

Comment:
On the client side, would it actually break anyone to enable this by default without a flag? It would be nice to avoid using a flag if we can.

(For testing purposes we'd still want to gate it with the `workerdExperimental` flag for now.)

Reply:
Definitely agree here. Really not convinced a separate compat flag is required here.

Reply:
https://github.com/cloudflare/workerd/compare/afa923039068570b760c372307bbeafd737f5a45..d47489133c59f4bd1dbe58000588f26620867bab

Switched to `workerdExperimental`, and then removed the compat flag piping since its not needed anymore https://github.com/cloudflare/workerd/compare/d47489133c59f4bd1dbe58000588f26620867bab..752f4e26b7381f239d44d44d65956e5b5540b0ef